lib/console.js

```diff
@@ -344,17 +344,26 @@ Console.prototype.table = function(tabularData, properties) {
   const getIndexArray = (length) => ArrayFrom({ length }, (_, i) => inspect(i));
 
   const mapIter = isMapIterator(tabularData);
+  let isKeyValue = false;
   if (mapIter)
-    tabularData = previewEntries(tabularData);
+    [ tabularData, isKeyValue ] = previewEntries(tabularData);
```

Comment:
The syntax makes me go 😬

Let's make it

```js
if (mapIter) {
  ;([tabularData, isKeyValue] = previewEntries(tabularData));
}
```

Reply:
you can have brackets or you can have a leading semicolon but my religion does not allow both 😉